client/src/pages/student.js

```diff
@@ -1,150 +1,131 @@
-import React, { useState } from 'react';
+import React, { Component, useState } from 'react';
 import CourseCheckbox from '../Components/CourseCheckbox';
 import { Link } from 'react-router-dom';
 
 
 
-const Student = () => {
-
-  let newfield = {
-      studentName: '',
-      classTimes:'',
-      hoursAvail:0,
-      coursePref:'',
-      facultyPref:'',
-      officeHours: 0
+class Student extends Component{
+  constructor(args) {
+    super(args);
+    this.state = {
+      
     }
-  //Creates state formFields and setFormfields variable and method
-  const [formFields, setFormFields] = useState([newfield])
-
-  //method to allow us to input data
-  const handleFormChange = (index, event) => {
-    let data = [...formFields];
-    data[index][event.target.name] = event.target.value;
-    setFormFields(data);
   }
 
-  //submit function
-  //VALIDTION FUNCTION NEEDS TO BE IMPLEMENTED HERE
-  const submit = (e) => {
-    e.preventDefault();
-    console.log(formFields)
+  render(){
+    console.log(this.props)
+    return (
+      <div>Testing page
```

Comment:
Creating a test page where all the text boxes for user input of student data are created.